app/routes/companyInterest/components/CompanyInterestPage.tsx

```diff
@@ -412,39 +237,7 @@ type CompanyInterestFormEntity = {
   companyToCompanyComment: string;
   companyPresentationComment: string;
   companyType: string;
-  officeInTrondheim: boolean;
-};
-
-type Props = {
-  allowedBdb: boolean;
-  onSubmit: (
-    arg0: CompanyInterestFormEntity,
-    arg1: boolean | null | undefined
-  ) => Promise<any>;
-  push: History['push'];
-  events: Array<Record<string, any>>;
-  companyCourseThemes: Array<Record<string, any>>;
-  semesters: Array<CompanySemesterEntity>;
-  otherOffers: Array<Record<string, any>>;
-  collaborations: Array<Record<string, any>>;
-  targetGrades: Array<Record<string, any>>;
-  participantRange: string;
-  edit: boolean;
-  interestForm: Record<string, any>;
-  companyInterest?: CompanyInterestEntity;
-  language: string;
-  comment: string;
-  courseComment: string;
-  breakfastTalkComment: string;
-  otherEventComment: string;
-  startupComment: string;
-  lunchPresentationComment: string;
-  bedexComment: string;
-  companyToCompanyComment: string;
-  companyPresentationComment: string;
-  companyType: string;
-  officeInTrondheim: boolean;
-  initialValues: any;
+  officeInTrondheim: 'yes' | 'no';
```

Comment:
Wouldn't this be simpler to treat as a boolean, than a stringified version of a boolean?

Reply:
Yes it was originally a boolean, but that caused a bug, so it was replaced with this. Here's the bug fix PR: https://github.com/webkom/lego-webapp/pull/3695